### ISSUE TITLE
Crate steal granting the bonus to faction its stolen from (at least for Destro)

### ISSUE THREAD

**Issue by Cuddlesthemighty** [WarEmu/WarBugs#20903]
TLDR: Order collecting Destruction faction crates awards Destruction exp and influence.

I was on my Black Orc in Black Crag while participating in RvR. I was at Rottenpike Ravine and destro had control of the point.  Two other players and I engaged a Dwarf Rune Priest.  After fighting for a bit we came to a stalemate and the other two players got bored and wandered off while I continued to engage.  They stunned me and proceeded to pick up crates which I then got a reward for.  I fought them for a bit it happened multiple times.

I don't think its a huge deal and would be pretty difficult to exploit but still not working as intended and I shouldn't get rewarded when the stunties punk me.
